src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
###AWAMlBG2b0Hum1nhz4KH###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_8@6
Complete the task associated to this TODO comment.

Reply:
simulating auto reply to outdated comment

Reply:
simulating 2nd auto reply to outdated comment